docs/Deploying-Pega-on-PKS.md

```diff
@@ -1,16 +1,16 @@
-Deploying Pega Platform on a PKS cluster
+Deploying Pega Platform on a TKGI cluster
```

Comment:
Is it worth noting in the beginning of this document that TKGI was formerly known as PKS?

Reply:
good thought...I added statement at the beginning: Deploy Pega Platform™ on a VMware Tanzu Kubernetes Grid Integrated Edition (TKGI) cluster using a PostgreSQL database you configure in Google Cloud Platform (GCP). TKGI was formerly knows Pivotal Container Service (PKS).